project/Dependencies.scala

```diff
@@ -42,7 +44,8 @@ object Dependencies {
     "org.fusesource.jansi" % "jansi" % "2.3.3",
     "org.jline" % "jline" % "3.20.0",
     "org.rogach" %% "scallop" % "4.0.3",
-    "net.sf.expectit" % "expectit-core" % "0.9.0" % "it,test"
+    "org.apache.logging.log4j" % "log4j-core" % "2.14.1",
+    "net.sf.expectit" % "expectit-core" % "0.9.0" % "it,test",
   )
```

Comment:
When shall we start sorting those dependencies alphabetically?

Reply:
I'd prefer we do that as a separate PR so we avoid mixing functional changes with style changes. I recently leanred about the .git-blame-ignore-revs file to ignore certain commits in a blame, which could be useful for ignore style-only changes, but only if there's no mixed in functional changes.

Reply:
I've created [DAFFODIL-2547](https://issues.apache.org/jira/browse/DAFFODIL-2547) to track sorting the depenencies.